sql_assign2.sql

```diff
@@ -0,0 +1,183 @@
+-- For this assignment, assume you have a database with the following table:
+-- CREATE TABLE Employees (
+--     EmployeeID INT PRIMARY KEY,
+--     FirstName VARCHAR(50),
+--     LastName VARCHAR(50),
+--     Department VARCHAR(50),
+--     Salary DECIMAL(10, 2),
+--     HireDate DATE
+-- );
+
+CREATE DATABASE Test;
+
+USE Test;
+
+CREATE TABLE Employees (
+    EmployeeID INT PRIMARY KEY,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Department VARCHAR(50),
+    Salary DECIMAL(10, 2),
+    HireDate DATE
+);
+
+INSERT INTO Employees (EmployeeID, FirstName, LastName, Department, Salary, HireDate) VALUES
+(1, 'John', 'Doe', 'Sales', 50000.00, '2022-01-15'),
+(2, 'Jane', 'Smith', 'Marketing', 60000.00, '2023-02-20'),
+(3, 'Michael', 'Johnson', 'HR', 55000.00, '2021-03-10'),
+(4, 'Emily', 'Davis', 'Finance', 70000.00, '2020-04-25'),
+(5, 'Daniel', 'Wilson', 'IT', 75000.00, '2019-05-30'),
+(6, 'Olivia', 'Brown', 'Sales', 48000.00, '2022-06-14'),
+(7, 'James', 'Jones', 'Marketing', 62000.00, '2023-07-19'),
+(8, 'Sophia', 'Garcia', 'HR', 53000.00, '2021-08-22'),
+(9, 'Liam', 'Martinez', 'Finance', 80000.00, '2020-09-05'),
+(10, 'Isabella', 'Hernandez', 'IT', 72000.00, '2019-10-12');
+
+
+-- Create a table students and insert names in malayalam
+CREATE TABLE Students (Stdid int, Stdname varchar(30));
+Alter Table Students Alter column Stdname nvarchar(30);
```

Comment:
Why creating the same column again ?

Reply:
By mistake I created table Students without using 'nvarchar', So altered it later